lib/matplotlib/bezier.py

```diff
@@ -316,7 +317,7 @@ def _f(xy):
 
 def get_cos_sin(x0, y0, x1, y1):
     dx, dy = x1 - x0, y1 - y0
-    d = (dx * dx + dy * dy) ** .5
+    d = (dx * dx + dy * dy) ** .5 or 1  # Account for divide by zero
```

Comment:
As a matter of style, I would much rather see more explicit handling of the zero case via an if statement:
```python
    if d == 0:
        return 0.0, 0.0
```